tabcmd/commands/datasources_and_workbooks/publish_command.py

```diff
@@ -66,18 +67,27 @@ def run_command(args):
         source = PublishCommand.get_filename_extension_if_tableau_type(logger, args.filename)
         logger.info(_("publish.status").format(args.filename))
         if source in ["twbx", "twb"]:
+            if args.thumbnail_group:
+                raise AttributeError("Generating thumbnails for a group is not yet implemented.")
+            if args.thumbnail_username and args.thumbnail_group:
+                raise AttributeError("Cannot specify both a user and group for thumbnails.")
+
             new_workbook = TSC.WorkbookItem(project_id, name=args.name, show_tabs=args.tabbed)
             try:
+                print(creds)
```

Comment:
This intentionally left in? Could it print a password?

Reply:
It should be a debug log, oops. 
But it won't print a password. Currently it will print a pretty unhelpful object reference, and when tsc is updated with a nice __repr__ method for the ConnectionCredentials object, it'll show the non-sensitive values, something like [ConnectionCredentials name="x" oAuth=False embedded=True] 
https://github.com/tableau/server-client-python/blob/master/tableauserverclient/models/connection_credentials.py